spiceaidocs/docs/features/caching/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Results Caching'
+sidebar_label: 'Results Caching'
+description: 'Learn how to use Spice in-memory caching of query results'
+sidebar_position: 5
+pagination_prev: null
+pagination_next: null
+---
+
+Spice supports in-memory LRU caching of query results, which can be enabled as follows. This can be used to prevent many redundant queries within a small time period and improve overall performance.
```

Comment:
```suggestion
Spice supports in-memory LRU caching of query results.  This can improve performance for redundant queries over a small period of time.
```